test/unittest/dds/subscriber/SubscriberTests.cpp

```diff
@@ -431,13 +431,13 @@ TEST(SubscriberTests, ChangeDefaultDataReaderQos)
     EXPECT_EQ(13, wqos.reliable_reader_qos().disable_positive_ACKs.duration.seconds);
     EXPECT_EQ(320u, wqos.reliable_reader_qos().disable_positive_ACKs.duration.nanosec);
     // .type_consistency
-    EXPECT_EQ(XML_DATA_REPRESENTATION, wqos.type_consistency().representation.m_value.at(0));
-    EXPECT_EQ(XCDR_DATA_REPRESENTATION, wqos.type_consistency().representation.m_value.at(1));
-    EXPECT_FALSE(wqos.type_consistency().type_consistency.m_ignore_sequence_bounds);
-    EXPECT_FALSE(wqos.type_consistency().type_consistency.m_ignore_string_bounds);
-    EXPECT_TRUE(wqos.type_consistency().type_consistency.m_ignore_member_names);
-    EXPECT_TRUE(wqos.type_consistency().type_consistency.m_prevent_type_widening);
-    EXPECT_TRUE(wqos.type_consistency().type_consistency.m_force_type_validation);
+    EXPECT_EQ(XML_DATA_REPRESENTATION, wqos.representation().m_value.at(0));
```

Comment:
I think we can go ahead and rename the `wqos` in this test with `rqos`, it is confusing as I interpreted them as `writer qos`